### DISCUSSION TITLE
One-Click Acknowledge Button for Inbox Documents

### DISCUSSION THREAD

**Discussion by Exordian** [jonaswinkler/paperless-ng#532] [Feature Requests]
As the title states, I typically follow the workflow:
- open first document in inbox
- check all properties/metadata
- remove "TODO" label
- click on "save and next"

since the "TODO" label has a special inbox flag, a button that just removes the inbox-flagged label, saves the document and jumps to the next one would be a nice UX addition. something like "Approve & Next"

**Comment by Miesjelangelo** [comment]
Nice one! or maybe behind the "save and next" button.
